test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetAddPackageTests.cs

```diff
@@ -613,5 +613,268 @@ public async Task AddPkg_WhenPackageSourceMappingConfiguredCanotInstallsPackages
             Assert.Contains($"Installed {packageX} {version} from {packageSource2}", result.AllOutput);
             Assert.Contains($"NU1100: Unable to resolve '{packageZ} (>= {version})' for 'net5.0'", result.AllOutput);
         }
+
+        [Fact]
+        public async Task AddPkgWithNoVersion_WhenProjectOnboardedToCPMAndNoPackagesExistInProps()
+        {
+            using var pathContext = new SimpleTestPathContext();
+
+            // Set up solution, and project
+            var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+            var projectName = "projectA";
+            var projectA = XPlatTestUtils.CreateProject(projectName, pathContext, "net5.0");
+
+            const string version1 = "1.0.0";
```

Comment:
nit: When you only use something in one spot, you don't need the property. 

Ideally tests are short and quick to read.

Reply:
Noted and resolved.